### DISCUSSION TITLE
How do I see all tray icons?

### DISCUSSION THREAD

**Discussion by medwatt** [i3/i3#4807] [Q&A]
I recently switched to i3 from KDE. In KDE, when I open programs like vlc, obe, etc., a small icon gets added to the icon tray on the KDE panel. Since moving to i3, these icons do not appear on the i3 bar. In other words, I no longer know whether these programs are running in the background. Actually, Zoom is the only program whose icon I can see. Is there any way to get the other icons?

**Comment by Jocadbz** [comment]
Strange issue. My OBS tray icon displays on i3 just fine. You are sure you didn't disabled any "Display icon" options on both apps? Because if Zoom icon shows up just fine, i don't believe there is an problem with i3 config file.
